n50/fetch_contigs.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import os
+import urllib.request
+from Bio import Entrez
+
+def main():
+    if len(sys.argv) != 7:
+        sys.exit("""
+Command usage: python fetch_contigs.py EMAIL NCBI_API_KEY GENUS_NAME SPECIES_NAME NUM_CONTIGS OUTPUT_DIRECTORY
```

Comment:
Usage should be updated for how you run this after it is installed as a python package.